README.md

```diff
@@ -95,55 +95,59 @@ fn main() {
         .run();
 }
 
+/// set up a simple 3D scene
 fn setup(
     mut commands: Commands,
     mut meshes: ResMut<Assets<Mesh>>,
     mut materials: ResMut<Assets<StandardMaterial>>,
 ) {
-    // Plane
+    // circular base
     commands.spawn((
         RigidBody::Static,
-        Collider::cuboid(8.0, 0.002, 8.0),
+        Collider::compound(vec![(
+            Position(Vec3::ZERO),
+            Rotation(Quat::from_rotation_x(-std::f32::consts::FRAC_PI_2)),
+            Collider::cylinder(0.002, 4.0),
+        )]),
         PbrBundle {
-            mesh: meshes.add(Plane3d::default().mesh().size(8.0, 8.0)),
-            material: materials.add(Color::rgb(0.3, 0.5, 0.3)),
+            mesh: meshes.add(Circle::new(4.0)),
+            material: materials.add(Color::WHITE),
+            transform: Transform::from_rotation(Quat::from_rotation_x(
+                -std::f32::consts::FRAC_PI_2,
+            )),
             ..default()
         },
     ));
-
-    // Cube
+    // cube
     commands.spawn((
         RigidBody::Dynamic,
         AngularVelocity(Vec3::new(2.5, 3.4, 1.6)),
         Collider::cuboid(1.0, 1.0, 1.0),
         PbrBundle {
-            mesh: meshes.add(Cuboid::default()),
-            material: materials.add(Color::rgb(0.8, 0.7, 0.6)),
+            mesh: meshes.add(Cuboid::new(1.0, 1.0, 1.0)),
+            material: materials.add(Color::rgb_u8(124, 144, 255)),
             transform: Transform::from_xyz(0.0, 4.0, 0.0),
             ..default()
         },
     ));
-
-    // Light
+    // light
```

Comment:
Nit, but I prefer to start comments with a capital letter, and have the extra newline between spawns, like what it was before. Bevy's *[own style guide](https://github.com/bevyengine/bevy/blob/main/.github/contributing/engine_style_guide.md)* says that comments should be capitalized, but it's contradicting itself in this example

```suggestion

    // Light
```

(this would of course apply to all other comments here too)